community/kernel/src/test/java/org/neo4j/kernel/api/index/IndexAccessorCompatibility.java

```diff
@@ -82,15 +82,15 @@ public void testIndexSeekByNumber() throws Exception
                 NodePropertyUpdate.add( 4L, PROPERTY_KEY_ID, 10.0, new long[]{1000} ),
                 NodePropertyUpdate.add( 5L, PROPERTY_KEY_ID, 100.0, new long[]{1000} ) ) );
 
-        assertThat( getAllNodesFromIndexSeekByNumber( 0, true, 10, false ), equalTo( asList( 2L, 3L ) ) );
-        assertThat( getAllNodesFromIndexSeekByNumber( 10, true, null, false ), equalTo( asList( 4L, 5L ) ) );
-        assertThat( getAllNodesFromIndexSeekByNumber( 10, false, null, true ), equalTo( singletonList( 5L ) ) );
-        assertThat( getAllNodesFromIndexSeekByNumber( 100, false, 0, true ), equalTo( EMPTY_LIST ) );
-        assertThat( getAllNodesFromIndexSeekByNumber( null, false, 5.5, false ), equalTo( asList( 1L, 2L ) ) );
-        assertThat( getAllNodesFromIndexSeekByNumber( null, true, 5.5, true ), equalTo( asList( 1L, 2L, 3L ) ) );
-        assertThat( getAllNodesFromIndexSeekByNumber( null, true, null, true ), equalTo( asList( 1L, 2L, 3L, 4L, 5L ) ) );
-        assertThat( getAllNodesFromIndexSeekByNumber( -5, false, 0, true ), equalTo( singletonList( 2L ) ) );
-        assertThat( getAllNodesFromIndexSeekByNumber( -5, false, 5.5, false ), equalTo( singletonList( 2L ) ) );
+        assertThat( getAllNodesFromInclusiveIndexSeekByNumber( 0, 10 ), equalTo( asList( 2L, 3L, 4L ) ) );
+        assertThat( getAllNodesFromInclusiveIndexSeekByNumber( 10, null ), equalTo( asList( 4L, 5L ) ) );
+        assertThat( getAllNodesFromInclusiveIndexSeekByNumber( 10, null ), equalTo( asList( 4L, 5L ) ) );
```

Comment:
These assertions could be compacted some since you removed the two boolean parameters. I'll fix when merging.